lib/Echidna/Solidity.hs

```diff
@@ -198,7 +219,6 @@ loadSpecified name cs = do
 
   -- Make sure everything is ready to use, then ship it
   when (null abi) $ throwM NoFuncs                              -- < ABI checks
-  neFuns <- maybe (throwM OnlyTests) pure (NE.nonEmpty funs)    -- <
```

Comment:
this no longer triggers the `OnlyTests` condition

Reply:
Fallback function will be always there, unless it is filtered. We discussed about handling this better in a future PR.